package.json

```diff
@@ -47,17 +47,17 @@
   },
   "devDependencies": {
     "@semantic-release/changelog": "^6.0.3",
-    "@semantic-release/commit-analyzer": "^11.1.0",
+    "@semantic-release/commit-analyzer": "^10.0.4",
```

Comment:
Don't think this downgrade is needed. Node v18 support is up to v11 on this. v12 is the one that drops Node v18 support

Reply:
v11 supports only `node >= 18.9.0` not the versions before that

Reply:
That's fine, you can set the engine version higher in package.json. Node v18 LTS is already on 18.18.x. Setting it to 18.9.x and higher is fine